index.html

```diff
@@ -220,19 +220,19 @@ <h3 class="languages">Languages</h3>
                     <ul class="language-list">
                         <li class="frame9">
                             <div class="lang">
-                                <img src="./images/Ellipse-1.png" alt="">
+                                <img src="./images/Ellipse-1.svg" alt="">
                                 <p>JavaSript</p>
                             </div>
                         </li>
                         <li class="frame10">
                             <div class="lang">
-                                <img src="./images/Ellipse-2.png" alt="">
+                                <img src="./images/Ellipse-2.svg" alt="">
                                 <p>HTML</p>
                             </div>
                         </li>
                         <li class="frame11">
                             <div class="lang">
-                                <img src="./images/Ellipse-3.png" alt="">
+                                <img src="./images/Ellipse-3.svg" alt="">
                                 <p>CSS</p>
                             </div>
                         </li>
```

Comment:
- [ ] As the previous reviewer mentioned, you should remove all commented code, kindly remove the commented code in line `243` and line `249`